index.js

```diff
@@ -74,7 +76,7 @@ module.exports = opts => {
 		}
 	});
 
-	if (Object.keys(cli).length) {
+	if (Object.keys(cli).length >= 0) {
```

Comment:
This check is moot as `.length` can only ever be 0 or more, so you're matching all possible scenarios. I think you meant `> 0`.